jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/dd/DDEditorServiceImpl.java

```diff
@@ -520,4 +521,12 @@ public boolean accepts(Path path) {
                             e.getMessage()));
         }
     }
+
+    private void addPredefinedInputsToContext(ParserContext context) {
+        context.addInput("ksession", Object.class);
+        context.addInput("taskService", Object.class);
+        context.addInput("runtimeManager", Object.class);
+        context.addInput("classLoader", Object.class);
+        context.addInput("entityManagerFactory", Object.class);
+    }
```

Comment:
Hi @tsurdilo, @mswiderski, @yurloc @triceo  sorry for late response, but this issues is doesn't looks fully fixed. I tried latest WorkBench build and when I follow Steps to Reproduce from [RHBPMS-4913](https://issues.jboss.org/browse/RHBPMS-4913) Jira issue I see this Validation error:
![screenshot from 2017-09-14 13-43-51](https://user-images.githubusercontent.com/1477262/30428130-c6bd1942-9952-11e7-8649-68c469beb5e0.png)
`Could not compile mvel expression 'new org.jbpm.process.workitem.bpmn2.ServiceTaskHandler(ksession, classLoader)'. This can be due to invalid syntax of missing classes - [Error: Failed to compileShared: 1 compilation error(s): 
 - (1,5) could not resolve constructor org.jbpm.process.workitem.bpmn2.ServiceTaskHandler[class java.lang.Object, class java.lang.Object]]
[Near : {... tem.bpmn2.ServiceTaskHandler(ksession, classLoader) ....}]
                                                               ^
[Line: 1, Column: 5]`
Probably, we need to add fields with not general Objects here. @tsurdilo, @mswiderski, what do you think? Can we fix it as well? Should we reopen [JBPM-6416](https://issues.jboss.org/browse/JBPM-6416) Jira issue?

Reply:
@hasys can you tell me how you built this because when i build master (jbpm-wb and kie workbench) I cannot even find the deployment descriptor option any more in the project settings. Something must have changed. So if you do see it maybe you are still testing with the older code?